starboard/android/shared/application_android.cc

```diff
@@ -412,7 +413,7 @@ void ApplicationAndroid::SendAndroidCommand(AndroidCommand::CommandType type,
   switch (type) {
     case AndroidCommand::kNativeWindowCreated:
     case AndroidCommand::kNativeWindowDestroyed:
-      while (native_window_ != data) {
+      while ((native_window_ != data) && !application_destroying_.load()) {
```

Comment:
there is an extra pair of parentheses

Reply:
I tend to add those on purpose for clarity.